Makefile

```
@@ -34,6 +34,8 @@ test-system:
 test-e2e:
 	poetry run pytest -n $(TEST_WORKER_COUNT) --dist loadscope tests/e2e
 
+install:
+	poetry install --all-extras
```

```suggestion
	poetry install
```
@izellevy the default install should definitely not be with all (redundant and maybe heavy) extras